### PR TITLE
Docs for RawTensor and BackendStatics

### PR DESCRIPTION
**Pull request by dsyme**
Simple enough

### CODE REVIEW

**Comment by dsyme** [comment]
@gbaydin I'll pull this, please review and adjust at your leisure

File: src/DiffSharp.Core/RawTensor.fs [hunk 1]
```diff
@@ -162,120 +225,313 @@ type RawTensor() =
...
+    /// Returns the 2D convolution of the tensor
+    abstract member Conv2D: kernel: RawTensor * strides: int[] * padding: int[] -> RawTensor
+
+    /// Returns the 3D convolution of the tensor
+    abstract member Conv3D: kernel: RawTensor * strides: int[] * padding: int[] -> RawTensor
+
+    /// Returns the element-wise negation of the tensor
     abstract member NegT : unit -> RawTensor
+
+    /// Returns the scalar tensor for the summation of all elements in the the tensor
```
Comment (kevmal) [inline src/DiffSharp.Core/RawTensor.fs:444]: "the the"

File: src/DiffSharp.Core/RawTensor.fs [hunk 2]
```diff
@@ -162,120 +225,313 @@ type RawTensor() =
...
+    /// Returns the tensor with the same values and the given dimension removed. The given dimension must be of size 1.
+    abstract member SqueezeT: dim: int -> RawTensor
+
+    /// Returns the tensor with the same values and a dimension of size 1 inserted before the given dimension.
+    abstract member UnsqueezeT: dim: int -> RawTensor
+
+    /// Returns the flip of the tensor along the given dimensions 
+    abstract member FlipT: dims: int[] -> RawTensor
+
+    /// Returns the dilation of the tensor using the given dilations paramaters
```
Comment (kevmal) [inline src/DiffSharp.Core/RawTensor.fs:465]: *parameters

File: src/DiffSharp.Core/RawTensor.fs [hunk 3]
```diff
@@ -162,120 +225,313 @@ type RawTensor() =
...
+    abstract member DilateT: dilations: int[] -> RawTensor
+
+    /// Returns the reverse of the dilation of the tensor using the given dilations paramaters
```
Comment (kevmal) [inline src/DiffSharp.Core/RawTensor.fs:468]: *parameters

**Comment by kevmal** [comment]
Sorry somehow didn't notice this was already merged when commenting... Put the noted changes in #180

**Pull request merged**